aws-codeartifact-repository/aws-codeartifact-repository.json

```
@@ -37,6 +37,26 @@
       "type": "string",
       "minLength": 1,
       "maxLength": 2048
+    },
+    "ExternalConnections": {
+      "description": "A list of external connections associated with the repository.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "Upstreams": {
+      "description": "A list of upstream repositories associated with the repository.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "PermissionsPolicyDocument": {
+      "description": "The access control resource policy on the provided repository.",
+      "type": "object",
+      "minLength": 2,
+      "maxLength": 5120
```

[`minLength`/`maxLength` is specifically for `string` types](https://json-schema.org/understanding-json-schema/reference/string.html#length)
[issue to catch these automatically](https://github.com/aws-cloudformation/cloudformation-cli/issues/414)
```suggestion
      "type": "object",
```

---

Thanks for calling this out: 

Created an issue: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeartifact/issues/60

Will cut a PR soon for this